test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -46,6 +46,21 @@ public void SetParameterByName_Element_CanSuccessfullySetMaterialByElement()
 
         }
 
+        [Test]
+        [TestModel(@".\element.rvt")]
+        public void CanSuccessfullyDeleteElement()
+        {
+            // Get element from document
+            var wall = ElementSelector.ByElementId(184176, true);
+            Assert.IsNotNull(wall);
+
+            // Delete element, confirming deletion success
+            Assert.IsTrue(wall.Delete());
+
+            // Verify element really deleted.
+            Assert.IsFalse(wall.Delete());
```

Comment:
This Double call of Delete() would throw an exception before the Assert.IsFalse called, then this test failed.
I think you can use RTF(https://github.com/DynamoDS/RevitTestFramework) to run the tests to make sure your tests is reliable. You can install the RTF or build by yourself.

Reply:
@ZiyunShang Good points. I am going to update the node api as follows for consistency with the internal revit api and some "safety" regarding the element getting deleted:

- I am going to make it a "static" routine so that there upon return from the delete, you are not in a member method and thus potential for issues accessing object data...
- match the internal api and return a list of the delete element Id's, or empty list upon failure to remove.